packages/admin-app/src/useToken.ts

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+
+/**
+ * A custom hook for an authentication token.  Currently uses session storage, thus token is lost when user
+ * closes tab.  To sacrifice security for convenience replace 'sessionStorage' with  'localStorage' and user
+ * will not need to log back in after closing tab.
```

Comment:
^ fyi decision here related to security to use sessionStorage over localStorage

Reply:
i heartily endorse